File: server/bcephal-reco/src/main/java/com/moriset/bcephal/reconciliation/domain/ReconciliationModelEditorData.java
/**
 * 
 */
package com.moriset.bcephal.reconciliation.domain;

import java.util.ArrayList;
import java.util.List;

import com.moriset.bcephal.domain.EditorData;
import com.moriset.bcephal.domain.Nameable;

import lombok.Data;
import lombok.EqualsAndHashCode;

/**
 * @author Joseph Wambo
 *
 */
@Data
@EqualsAndHashCode(callSuper=true)
public class ReconciliationModelEditorData extends EditorData<ReconciliationModel> {

	private List<Nameable> sequences;
	
	private Long debitCreditAttributeId;

	private String debitValue;

	private String creditValue;
	
	public List<Nameable> matGrids;
	
	public ReconciliationModelEditorData() {
		sequences = new ArrayList<>(0);
		matGrids = new ArrayList<>(0);
	}
	
}

File: client/bcephal-client-model/Reconciliation/ReconciliationModelEditorData.cs
using Bcephal.Models.Base;
using System;
using System.Collections.Generic;
using System.Collections.ObjectModel;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Bcephal.Models.Reconciliation
{
    public class ReconciliationModelEditorData : EditorData<ReconciliationModel>
    {

        public ObservableCollection<Nameable> Sequences { get; set; }

        public long? DebitCreditAttributeId { get; set; }

        public string DebitValue { get; set; }

        public string CreditValue { get; set; }

    }
}
